20-packages.ks

```diff
@@ -77,9 +77,14 @@ e2fsprogs
 bzip2
 system-storage-manager
 
-#
-# Packages to Remove
-#
+######################
+# Packages to Remove #
+######################
+
+# Red Hat Enteprise Linux subscription tool
+-subscription-manager
```

Comment:
We used to have it downstream but now it needs more dependencies so let's get rid of it.